modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -55,10 +77,53 @@ function inope_policy_review_tab_title_callback($node) {
  *   policy nodes.
  */
 function inope_policy_review_page($node) {
-  // Create a build array to be displayed on the page.
-  $build = array(
-    'inope_policy_review_form' => drupal_get_form('inope_policy_review_form', $node),
-  );
+  global $user;
+  $account = $user;
+  $build = array();
+  $policy_review_id = NULL;
+
+  // Get the parameters passed in the url and if possible set policy_review_id.
+  $get_params = drupal_get_query_parameters();
+  if (!empty($get_params['id'])) {
+    $policy_review_id = $get_params['id'];
+  }
+
+  // Retreive all notes related to this node and account.
+  $results = inope_policy_review_select_query($node, $account);
+
+  // If the active user has previously submitted something only display a
+  // message.
+  if (empty($results)) {
+    // If no results build a blank form.
+    $build['inope_policy_review_form']
+      = drupal_get_form('inope_policy_review_form', $node);
```

Comment:
I recently found out that executable code doesn't have length compliance and it is just on the comments.  But at the time the line was too long for me to view so I sent it to a new line in a code compliance format (equals sign is on the second line)